### PR TITLE
Collins-shell fix dependencies

### PR DESCRIPTION
**Pull request by Primer42**
A few changes here:
- Remove `jewler` and `rubygems-update` dependencies
- Remove weird backwards compatible jewler code from gemspec file 
- Use gemspec in Gemfile rather than declaring all dependencies twice

@tumblr/collins

### CODE REVIEW

File: support/ruby/collins-shell/collins_shell.gemspec [hunk 1]
```diff
@@ -1,95 +1,70 @@
...
+    'lib/collins_shell/util.rb',
+    'lib/collins_shell/util/asset_printer.rb',
+    'lib/collins_shell/util/asset_stache.rb',
+    'lib/collins_shell/util/log_printer.rb',
+    'lib/collins_shell/util/printer_util.rb'
   ]
-  s.homepage = "https://github.com/tumblr/collins/tree/master/support/ruby/collins-shell"
-  s.licenses = ["APL 2.0"]
-  s.require_paths = ["lib"]
-  s.rubygems_version = "1.8.23"
```
Comment (Primer42) [inline support/ruby/collins-shell/collins_shell.gemspec]: Docs say that this should never be set by hand, so I deleted it
http://guides.rubygems.org/specification-reference/#rubygems_version

**Comment by komapa** [comment]
:bullettrain_front: LGTM

**Comment by roymarantz** [comment]
:+1:

**Comment by Primer42** [comment]
The scala tests failed, but all the ruby tests passed, so this is safe to go out.

**Pull request merged**